src/main/java/net/mcreator/preferences/data/BackupsSection.java

```diff
@@ -21,7 +21,9 @@
 
 import net.mcreator.preferences.PreferencesSection;
 import net.mcreator.preferences.entries.BooleanEntry;
+import net.mcreator.preferences.entries.FileEntry;

```

Comment:
Seems unrelated to PR

Reply:
I forgot to remove them after creating the section 😅 Sorry